recipes/nyxus/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "nyxus" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sameeul/nyxus/archive/refs/heads/remove_submodules.zip
+  sha256: 0bb4df723a669d6ab0d8358b8a123e71094e12ba7cdd6273026aad54044c35ac
+
+build:
+  number: 0
+  skip: true  # [py<36]
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - setuptools
+    - git
+    - z5py 
+    - libtiff 
+    - boost 
+    - nlohmann_json 
+    - blosc 
+    - pybind11 
+    - xtensor 
+    - xsimd 
+    - numpy
+    - pandas 
+    - m2-patch  # [win]
+
+  host:
+    - python {{ PY_VER }}
+
+  run:
+    - python {{ PY_VER }}
+    - libtiff 
+    - blosc 
+    - numpy
```

Comment:
if it actually builds against numpy, there are some tricks:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy